st2client/tests/base.py

```diff
@@ -86,6 +86,8 @@ def setUp(self):
             if var in os.environ:
                 del os.environ[var]
 
+        os.environ['ST2_SKIP_CLI_CONFIG'] = '1'
```

Comment:
Is this needed only for tests? I am not seeing a genuine use case outside tests?